content/02.introduction.md

```diff
@@ -47,46 +46,96 @@ The transmission and mortality rate estimations of COVID-19 remain to be determi
 
 Among known coronaviruses, SARS-CoV-2 has the closest overall sequence similarity to RaTG13 (~96%) found in a *Rhinolophus affinis* bat [@doi:10.1038/s41586-020-2012-7], while the receptor binding domain (RBD) is highly similar to that of viruses found in pangolins [@doi:10.1101/2020.02.19.950253].
 This suggests that SARS-CoV-2 may have originated in viral reservoirs of similar hosts; however, current evidence cannot discriminate an origin of the virus before or after zoonotic transfer to humans [@doi:10.1038/s41591-020-0820-9].
 
-### COVID-19: Mechanisms and Presentation
-
-#### Immunological Mechanisms of Coronavirus-driven Disease in Humans
-
-Coronaviruses are known to cause respiratory illnesses in humans through the following possible mechanisms...[Summarize relevant mechanisms for cell entry & address evidence for/against ACE2 being important, etc.]
+The viral spike protein, _S_, plays a critical role in viral replication.
+There are two domains: the N-terminal (S1 )domain used for binding and the C-terminal (S2) domain for fusion [@doi:10.3390/v4061011].
+The S1 domain evolves more rapidly than the S2 domain, and different coronaviruses bind to difference receptors.
+Betacoronaviruses are known to bind to CEACAM1, Neu 5,9 Ac2, and ACE2 [@doi:10.3390/v4061011].
+
+### COVID-19: Infection and Presentation
+
+#### Pathogenic Mechanisms of Coronaviruses
+
+Until more information becomes available about the pathogenesis of SARS-CoV-2 specifically, insight can be gained by examining how the human body responds to viral pathogens generally, how infection from related viruses proceeds, and how this information intersects with what is known about COVID-19.
+Following exposure to a pathogen such as a virus, pathogenesis involves three major components: entry, replication, and spread [@isbn:0-9631172-1-1 {chap. 45}]. 
+The human immune response is double-pronged, with the innate and the adaptive immune response.
+Through innate immunity, the body can rapidly recognize and attack pathogens and infected cells. Cells from innate immunity also pass critical information to prime the adaptive immune response.
+Adaptive immunity can be broadly classified into humoral and cell-mediated responses. 
+These are unique from innate immune responses because they target specific protein sequences from pathogens. 
+The humoral response includes soluble proteins and antibodies that can neutralize and eliminate pathogens in circulation that have not yet entered cells, and can also mediate killing of pathogen-infected cells. 
+Cell-mediated responses include the killing of pathogen-infected cells by cytotoxic CD8+ T cells, and the secretion of cytokines to drive inflammation and support immune cell activities.
+The antibody-mediated response works to prevent or reduce entry into cells, while the cell-mediated response works to stop the pathogen's replication and spread.
```

Comment:
A couple of sentences which we could add here (or elsewhere if you find better place) that go along with describing major general points on immune system:
Thus, pathogens infects hosts and hosts have immune defenses and responses; furthermore, pathogens have developed ways to evade the immune system (as well as, in general, various ways to manipulate the host).
Other general complexities to consider include that disease symptoms can be due to a pathogen or due to the reaction of the immune system to the pathogen and that the host immune system can defend against or fight infection but it can also sometimes cause too much damage to the host [doi: 978-0815332183] [doi: 9780071283663].